### PR TITLE
Karolina social media

### PR DESCRIPTION
**Pull request by moharso**

### CODE REVIEW

File: src/components/DarkLightButton/DarkLightButton.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+// import { ThemeContext } from '../../context/ThemeContext';
```
Comment (neringa1991) [inline src/components/DarkLightButton/DarkLightButton.tsx:2]: As dicussed, let's remove all the comments once we finish with our parts :)

File: src/components/addCaption/AddCaption.tsx [hunk 2]
```diff
@@ -102,20 +102,14 @@ const AddCaption = ({onClickSubmit, selectedImage}: any) => {
...
-            <NavLink
-              to="/"
-              // className="flex items-center font-Bold leading-none max-w-full transition duration-100 outline-none bg-sky-300 focus:outline-none justify-center bg-icoBlue text-white hover:bg-sky-400 focus:bg-icoBlue-300 focus:ring-2 focus:ring-icoBlue-200 focus:ring-offset-2 focus:text-white px-10 py-5 min-h-14 min-w-150 text-sm rounded-md disabled:bg-icoGray-300 dark:disabled:bg-white/5 disabled:text-icoGray-400 dark:disabled:text-icoDarkMode-wolf disabled:cursor-not-allowed"
-            >
-              ◀ Previous: Calendar
-            </NavLink>
-            <button
-              // className="flex items-center font-Bold leading-none max-w-full transition duration-100 outline-none bg-sky-500 focus:outline-none justify-center bg-icoBlue text-white hover:bg-sky-400 focus:bg-icoBlue-300 focus:ring-2 focus:ring-icoBlue-200 focus:ring-offset-2 focus:text-white px-10 py-5 min-h-14 min-w-150 text-sm rounded-md disabled:bg-icoGray-300 dark:disabled:bg-white/5 disabled:text-icoGray-400 dark:disabled:text-icoDarkMode-wolf disabled:cursor-not-allowed"
+            <NavButton buttonText="<-return" to="/"></NavButton>
```
Comment (neringa1991) [inline src/components/addCaption/AddCaption.tsx:105]: Agree with this change and thanks for adding reusable component. I will change buttons to reusable once in my part, too

File: src/components/data/data.ts [hunk 3]
```diff
@@ -68,13 +68,13 @@ export const myEventsList = [
 
 const myUsersList = [
   {
-    userName: "neringa",
+    userName: "Neringa",
```
Comment (neringa1991) [inline src/components/data/data.ts:71]: these usernames indeed make more sense :)

**Review by neringa1991 — APPROVED** [review]

File: src/components/DarkLightButton/DarkLightButton.tsx (hunk 1 above)
Comment (moharso) [reply src/components/DarkLightButton/DarkLightButton.tsx:2]: I will use this one for sure, it's just not working now, so I commented that part :D

**Pull request merged**